models/Shape/SquareFactory.java

```diff
@@ -1,5 +1,33 @@
 package models.Shape;
 
 final public class SquareFactory {
+    public double height;
+    public double width;
+    public double area ;
```

Comment:
rm space after area for consistency. Is there a specific reason these are public?